rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java

```diff
@@ -54,28 +71,133 @@
 import org.hamcrest.MatcherAssert;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
-
 import java.math.BigInteger;
 import java.nio.charset.StandardCharsets;
 import java.time.Instant;
 import java.util.*;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.CoreMatchers.nullValue;
-import static org.hamcrest.Matchers.*;
-import static org.junit.jupiter.api.Assertions.assertArrayEquals;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotEquals;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.fail;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
 class BridgeSerializationUtilsTest {
 
+    private static final NetworkParameters NETWORK_PARAMETERS = NetworkParameters.fromID(NetworkParameters.ID_REGTEST);
+    private static final Address ADDRESS = Address.fromBase58(NETWORK_PARAMETERS, "mvc8mwDcdLEq2jGqrL43Ub3sxTR13tB8LL");
+    private static final Address OTHER_ADDRESS = Address.fromBase58(NETWORK_PARAMETERS, "n3CaAPu2PR7FDdGK8tFwe8thr7hV7zz599");
+
+    @Test
+    void serializeAndDeserializeSingleEntry_whenValidData_shouldReturnEqualResults() {
+        // Arrange
+        BtcTransaction input = new BtcTransaction(NETWORK_PARAMETERS);
```

Comment:
calling it `input` is kind of confusing to me, wdyt about `previousTransaction`?
Also, i think it could be defined at class level to avoid code duplication

Reply:
> calling it input is kind of confusing to me, wdyt about previousTransaction?

agree, updated

> Also, i think it could be defined at class level to avoid code duplication

I would keep it as is since it is only being used in two tests. Also made sure of using different outputs to avoid code dedup.